app/moves/views/list-by-status.njk

```diff
@@ -0,0 +1,71 @@
+{% extends "./_layout.njk" %}
+
+  {% block pageHeader %}
+    <header class="govuk-grid-row">
+      <div class="govuk-grid-column-full">
+        <span class="govuk-caption-xl">{{ t(pageTitle) }}</span>
+
+        {% block dateTitle %}
+          <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">
+            {{ appTime({
+              datetime: datetime,
+              text: displayDate
+            }) }}
+          </h1>
+        {% endblock %}
+
+        {% block dateNavigation %}
```

Comment:
It feels like there's a few things in this template that could be inherited from a parent layout for the other listing views?

You could add some blocks to that layout to help you inject content, like the filter, where needed rather than having to duplicate the pagination component, header markup, etc.

Reply:
There's some duplication, indeed -- mostly because I need `govuk-grid-column-full` and the other layouts don't. Is there a way to inject a class that doesn't feel contrived?